app/components/ui/atoms/Textarea.js

```diff
@@ -45,14 +41,28 @@ class Textarea extends React.Component {
     this.inputId = `textarea-${Math.round(Math.random() * 1e12).toString(36)}`
   }
   render() {
-    const { label, value = '', readonly, ...props } = this.props
+    const { label, readonly, type = 'text', value = '', ...props } = this.props
     return (
-      <Root>
-        {label && <Label htmlFor={this.inputId}>{label}</Label>}
-        <Input id={this.inputId} readOnly={readonly} value={value} {...props} />
-      </Root>
+      <Flex flexDirection="column">
+        <Box mb={1}>
+          <Label htmlFor={this.inputId}>{label}</Label>
+        </Box>
+        <Box>
+          <Input
+            id={this.inputId}
+            readOnly={readonly}
+            type={type}
```

Comment:
What does `type` do on a textarea?

Reply:
Nothing at all. Thanks @tamlyn 

Made the change :+1: